language/en/INGAME.php

```diff
@@ -275,10 +275,10 @@
 $LNG['fl_mission']							= 'Mission';
 $LNG['fl_ammount']							= 'Number';
 $LNG['fl_beginning']						= 'Start';
-$LNG['fl_departure']						= 'Arrival(Destination)';
+$LNG['fl_departure']						= 'Arrival (Destination)';
 $LNG['fl_destiny']							= 'Destination';
 $LNG['fl_objective']						= 'Target';
-$LNG['fl_arrival']							= 'Arrival(Back)';
+$LNG['fl_arrival']							= 'Return Arrival';
```

Comment:
This is tricky. fl_departure is "Arrival (Destination)". We need something with round brackets to keep convention, too.

Reply:
Good call. I'll ponder this one.